### PR TITLE
Set Placement as the default in the policy wizards

### PR DESCRIPTION
**Pull request by mprahl**
Relates:
https://issues.redhat.com/browse/ACM-4063

### CODE REVIEW

File: frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx [hunk 1]
```diff
@@ -74,19 +68,66 @@ describe('Create Policy Page', () => {
       nockCreate(mockPolicySets[2]),
     ]
 
-    const placementRuleNock = [
-      nockCreate(mockPlacementRules[1], undefined, 201, { dryRun: 'All' }), // DRY RUN
-      nockCreate(mockPlacementRules[1]),
+    const mockPlacement: Placement = {
```
Comment (dhaiducek) [inline frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx:71]: Nit: I'd prefer these be kept in the `governance.sharedMocks.tsx` file to keep with existing conventions.

File: frontend/src/wizards/Policy/PolicyWizard.tsx [hunk 2]
```diff
@@ -177,7 +176,7 @@ export function PolicyWizard(props: {
           existingClusterSetBindings={props.clusterSetBindings}
           bindingSubjectKind={PolicyKind}
           bindingSubjectApiGroup={PolicyApiGroup}
-          defaultPlacementKind={props.defaultPlacementKind ?? PlacementRuleKind}
+          defaultPlacementKind={PlacementKind}
```
Comment (dhaiducek) [inline frontend/src/wizards/Policy/PolicyWizard.tsx:179]: Optional/Future: It looks to me like GRC is the only set of wizards that uses the `PlacementSection` component, so we can get rid of this property entirely here and make changes there.

**Review by dhaiducek — COMMENTED** [review]
Looks good--some minor comments to consider.

File: frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx (hunk 1 above)
Comment (dhaiducek) [reply frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx:71]: So...do you want to add a Placement/PlacementBinding mock generation function to the sharedMocks file to use here? 🙂

**Review by dhaiducek — APPROVED** [review]
/hold in case the mock generation function should be implemented in this PR

File: frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx (hunk 1 above)
Comment (mprahl) [reply frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx:71]: For the sake of time, I think I'll defer that for now. :smile:

**Pull request merged**